### DISCUSSION TITLE
2024.03.11(월)

### DISCUSSION THREAD

**Discussion by Crack-Egg** [Party-Time-Job/Party-Time-Job#28] [회의록]
## 1. 전체 일정

계획한 전체 일정에서 전체적으로 1~2일 더 미뤄질 수 있음.

일찍 끝나면 디자인, 추가 기능 작업 예정.

⇒ 재명님은 전체 일정 작성한대로 진행되고, 나머지 팀원들은 1~2일 정도 늦춰질 수 있고, 일정 끝나면 추가 기능 작업하고, 없을 경우 다른 팀원 도와주기 또는 디자인 변경 작업하기

## 2. 디자인

UI는 크게 변경하지는 않고, 필요한 부분만 변경하는 방향으로 진행(UI 수정하게 되면 조금 데드라인까지 완성이 어려울 수 있다고 판단)

로고 → 팀끼리 만들지(시간) vs 외주(금액)

⇒ 각자 로고에 대한 방향, 기초적인 스타일 등 준비해오기!

## 3. 깃허브

`Wiki` → 규칙 및 컨벤션 정리

`Projects` → 전체 일정(대기중, 진행중, 완료중)

`Issues` → 할 일 생성(옵션 선택)

`Discusstion` → 회의록 및 회고록 작성(회고는 1주일에 한번 진행) → 팀 회고는 토요일 저녁까지(22:00) 제출하기!

## 4. PR 요청

PR올릴 때는 관련이슈에 Issues를 지정해주기 때문에 Projects 옵션은 따로 지정해주지 않아도 된다. (Reviewers, Assignee만 설정하면 된다)

**Comment by DanteSnow** [comment]
추가 내용

1. 전체 일정 및 차후 계획
 - 전체 작업 일정 이틀 미뤄짐. (재명님 제외)
 - 작업하면서 추가 기능 구상하기
 - 작업 끝난 인원은 바로 추가 기능 작업 착수
 - 구상한 추가 기능 없을 시, 작업 중인 팀원에게 합류

2. 디자인
 - 로고를 우선으로 구상하고, 나머지는 로고에 맞춰서 바리에이션
 - 오늘 안으로 원하는 디자인 방향 및 로고 레퍼런스 하나 이상 정해오기 (해당 내용들로 내일 디자인 회의 예정)

3. 리코일 사용법 공유
 - 오늘 안으로 디스코드 자료공유 스레드에 업로드 예정

4. 팀 회고
 - 토요일 오후 10시까지 각자 팀 회고 작성해서 팀장에게 전달
 - 양식 자유
